add uuid by objectUUID as it was missing

Rob9315 opened the pull request:
I tried reading the UUID of the entity of a dropped Item and it was undefined. this should fix that

wvffle commented on lib/plugins/entities.js:
```diff
@@ -189,6 +189,7 @@ function inject (bot, { version }) {
     }
 
     entity.uuid = packet.entityUUID
+    if (packet.objectUUID) entity.uuid = packet.objectUUID
```
```js
    if (packet.objectUUID) {
      entity.uuid = packet.objectUUID
    }
```

also move it either above or below the assigments with newlines around the if

Rob9315 commented:
so now I realized that the "spawn_entity" packet does not have an entityUUID property, it is always objectUUID so now I removed the if statement and just replaced the entityUUID by objectUUID

wvffle commented:
Are you sure that previous versions have `objectUUID` instead of `entityUUID` as well?
